docs/deployment/README.md

```diff
@@ -23,10 +26,7 @@ This deployment document is current for the following component versions:
 The process involves:
 
 * Creating [AWS S3 Buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html).
-* Using [Kes](http://devseed.com/kes/) to transform kes templates (`cloudformation.template.yml`) into [AWS CloudFormation](https://aws.amazon.com/cloudformation/getting-started/) stack templates (`cloudformation.yml`) that are then deployed to AWS.
-* Before deploying the Cumulus software, a CloudFormation stack is deployed that creates necessary [IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html) via the `iam` stack.
-* Database resources are configured and deployed via the `db` stack.
-* The Cumulus software is configured and deployed via the `app` stack.
+* Using [Terraform](https://www.terraform.io) to deploy resources to AWS
```

Comment:
We should probably  include something about having access to a CMA layer here

Reply:
Also, don't we need to include creating the locks table?

Reply:
Yeah, that should probably be mentioned up here

Reply:
I added some additional steps to cover everything involved. Though upon further review this kind of reads like a TOC. In the spirit of not repeating ourselves, do we need it?

Reply:
I think in this instance the repetition is fine, as it's telling them what we're telling them (framing the rest of the information).    It's not repeating technical information that requires cross-linked maintenance, it's self-referential to this doc.     I can see the argument for not doing it as well though.

Reply:
I think 65592de3b is sufficient for this cookbook - how's that look?